### PR TITLE
Update FAQ to add Smart Contract questions

### PR DESCRIPTION
**Pull request by avsavsavs**
Updates the FAQ like we had updated in the a conflicting branch (https://github.com/gnosis/cowswap/pull/1716/files)

Includes SC questions & SC operations in the table

There wasn't an issue but it fixes pull 1716 as this request was done in the wrong branch

### CODE REVIEW

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -408,16 +408,38 @@ export default function Faq() {
...
-            working to make this a possibility and support will be added soon.
+            CowSwap uses offchain signatures to offer gasless orders, additionally has support for smart contract
+            wallets by using some alternative signing method called{' '}
+            <LinkScrollable href={'#what-is-presign'}>pre-sign</LinkScrollable>.
           </p>
           <p>
-            Nevertheless, even if your wallet is not an SC wallet, it might be unsupported in some cases. Not all
-            wallets implement the necessary signing methods from EIP712 standard. If that is the case for you, reach out
-            to your wallet developers and ask for it.
+            Smart Contract (SC) are supported by using pre-sign method, however for normal wallets (EOA) CowSwap
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:417]:
```suggestion
            Smart Contracts (SC) are supported by using pre-sign method, however for normal wallets (EOA) CowSwap
```

File: src/custom/pages/Faq/index.tsx [hunk 2]
```diff
@@ -408,16 +408,38 @@ export default function Faq() {
...
+            requires the wallet to support offchain-signing (EIP712 standard).
+          </p>
+          <p>
+            Some wallets have reported issues with this offchain-signing, so they disabled it. If that is the case for
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:421]:
```suggestion
            Some wallets have issues with offchain-signing, so we have disabled them. If that is the case for
```

File: src/custom/pages/Faq/index.tsx [hunk 3]
```diff
@@ -408,16 +408,38 @@ export default function Faq() {
...
+          <h3 id="smart-contract-support">Are smart contract integrations supported?</h3>
+          <p>Yes! Any smart contract can trade in CowSwap by using the signing method called pre-sign.</p>
+          <p>
+            pre-sign is a protocol operation that can be invoked by any contract. The operation has a single parameter
+            that is the &quot;orderId&quot; which identifies the order that the smart contract is approving. Once the
+            smart contract pre-signs an order, it becomes automatically tradable, therefore solvers will start
+            considering the order for their settlement solutions.
+          </p>
+          <p>
+            In the future, the protocol could provide EIP712 support for off-chain signing also for smart contracts,
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:433]: I believe this was mentioned elsewhere. It's not EIP712 for SC. Is it https://eips.ethereum.org/EIPS/eip-2771 maybe?

**Comment by anxolin** [comment]
@alfetopito this PR was created in the wrong branch. should be in `release/1.5`. I tried to rebase this one, but ther's conflicts cause we work in the similar things. The easiest is to re-create this starting from that branch

**Comment by alfetopito** [comment]
> @alfetopito this PR was created in the wrong branch. should be in `release/1.5`. I tried to rebase this one, but there's conflicts cause we work in the similar things.
> 
> @avsavsavs The easiest is to re-create this starting from that branch

ohhh I see, it's `1.5` instead of `release/1.5`

**Pull request closed without merging**